plugins/source/aws/resources/services/dynamodb/global_tables.go

```diff
@@ -0,0 +1,95 @@
+package dynamodb
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func GlobalTables() *schema.Table {
+	tableName := "aws_dynamodb_global_tables"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_GlobalTableDescription.html`,
+		Resolver:            fetchGlobalTables,
+		PreResourceResolver: getGlobalTable,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "dynamodb"),
+		Transform:           transformers.TransformWithStruct(&types.GlobalTableDescription{}),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("GlobalTableArn"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "tags",
+				Type:     schema.TypeJSON,
+				Resolver: resolveDynamodbGlobalTableTags,
+			},
+		},
+	}
+}
+
+func fetchGlobalTables(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Dynamodb
+
+	config := dynamodb.ListGlobalTablesInput{
+		RegionName: aws.String(c.Region),
+	}
+	for {
+		output, err := svc.ListGlobalTables(ctx, &config)
```

Comment:
Isn't it [legacy](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ListGlobalTables.html)?

Even more, I'd argue that we need to use [TableDescription.GlobalTableVersion](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TableDescription.html) field from [DescribeTable](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_DescribeTable.html) output to determine whether we need to obtain global info for the table via [API_DescribeTableReplicaAutoScaling](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/DescribeTableReplicaAutoScaling.html) call.

So, there should be `aws_dynamodb_table_replica_auto_scaling` relation for `aws_dynamodb_tables` that would be populated for tables with `GlobalTableVersion != ""` (better yet, for `GlobalTableVersion == "2019.11.21"`).

Reply:
While I don't disagree we need to add that data, I think it falls outside the scope of this pr as the old GlobalTable is still a valid resource that users use

Reply:
I suggest adding the same notice as AWS has (about this being only for legacy API) in the description